### PR TITLE
Topic Test [12816]

### PR DESCRIPTION
**Pull request by jparisu**
Merge after #25

### CODE REVIEW

File: src/cpp/types/topic/DatabrokerTopic.cpp [hunk 1]
```diff
@@ -65,9 +65,24 @@ bool DatabrokerTopic::operator <(
...
+        else
+        {
+            return false;
+        }
     }
 }
 
+std::ostream &operator<<(std::ostream &os, const DatabrokerTopic& a)
+{
+    os << "{<" << a.topic_name() << ">:<" << a.topic_type() << ">}";
```
Comment (rsanchez15) [inline src/cpp/types/topic/DatabrokerTopic.cpp:83]: I'd remove the `<>` as it indicates that there should be something else there.
```suggestion
    os << "{" << a.topic_name() << ":" << a.topic_type() << "}";
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: src/cpp/types/topic/DatabrokerTopic.cpp (hunk 1 above)
Comment (jparisu) [reply src/cpp/types/topic/DatabrokerTopic.cpp:83]: I though about it, but there are topics and especially types that use `:` i.e. `std::msg::string`

Comment (rsanchez15) [reply src/cpp/types/topic/DatabrokerTopic.cpp:83]: Okay so maybe lets use
```cpp
os << "{" << a.topic_name() << ", " << a.topic_type() << "}";
```

Comment (jparisu) [reply src/cpp/types/topic/DatabrokerTopic.cpp]: :+1:

**Review by rsanchez15 — APPROVED** [review]

**Comment by rsanchez15** [comment]
Closed in favor of #31

**Comment by rsanchez15** [comment]
Closed in favor of #31

**Pull request closed without merging**